### PR TITLE
Add translation documentation for Weblate

### PR DESCRIPTION
**Pull request by joshuaboniface**

### CODE REVIEW

File: docs/contributor-docs/contributing.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Documentation is incredibly helpful! All these docs are written using `mkdocs` v
 
 ### Translations
 
-We're still working to set up a translation system that will help automate this, but for now translations occur in the frontend code. This is a complex process so please ask for help if you wish to assist right now.
+If you're interested in helping to translate Jellyfin into your local language, we use [Weblate](https://weblate.org/en/) running at [translate.jellyfin.org](https://translate.jellyfin.org) to handle translations. These are collected in the `translations` branch of the [jellyfin-web](https://github.com/jellyfin/jellyfin-web) repository and are merged into master before each release.
```
Comment (JustAMan) [inline docs/contributor-docs/contributing.md]:
```suggestion
If you're interested in helping to translate Jellyfin into your local language, we use [Weblate](https://weblate.org/en/) running at [translate.jellyfin.org](https://translate.jellyfin.org) to handle translations. These are collected in the [translations](https://github.com/jellyfin/jellyfin-web/tree/translations) branch of the [jellyfin-web](https://github.com/jellyfin/jellyfin-web) repository and are merged into master before each release.
```
might also want to indicate that this is for Web UI only _for now_, and more are to come.

Comment (joshuaboniface) [reply docs/contributor-docs/contributing.md]: I tweaked this up a bit more now that it's not just `jellyfin-web` being translated and made it repo-generic..

File: docs/index.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 <br/><br/>
 <a href="https://github.com/jellyfin/jellyfin"><img alt="GPL 2.0 License" src="https://img.shields.io/github/license/jellyfin/jellyfin.svg"></a>
 <a href="https://github.com/jellyfin/jellyfin/releases"><img alt="Current Release" src="https://img.shields.io/github/release/jellyfin/jellyfin.svg"></a>
+<a href="https://translate.jellyfin.org/engage/jellyfin-web/?utm_source=widget"><img src="https://translate.jellyfin.org/widgets/jellyfin-web/-/svg-badge.svg" alt="Translation status" /></a>
```
Comment (anthonylavado) [inline docs/index.md]: The badge doesn't load, 404

Comment (joshuaboniface) [reply docs/index.md]: Oh, you got it before I pushed up the fix ;-)

**Pull request merged**